sdk/python/feast/infra/online_stores/redis.py

```diff
@@ -160,9 +163,13 @@ def update(
             entities_to_keep: Entities to keep
             partial: Whether to do a partial update
         """
+        online_store_config = config.online_store
+
+        assert isinstance(online_store_config, RedisOnlineStoreConfig)
 
-        for table in tables_to_delete:
-            self.delete_table(config, table)
+        if online_store_config.full_scan_for_deletion:
```

Comment:
the old implementation deletes tables by default.
should `full_scan_for_deletion`'s default value be `True`?

Reply:
As you said, I set default to True for compatibility with older implementations.
If compatibility is not enforced, I think False would be better.

Reply:
@breno-costa Changed to default True to ensure backward compatibility